test/base/docker-compose.yml

```diff
@@ -28,7 +28,7 @@ services:
       - SW_HEALTH_CHECKER=default
       - SW_TELEMETRY=prometheus
     healthcheck:
-      test: [ "CMD", "sh", "-c", "nc -zn 127.0.0.1 11800" ]
+      test: [ "CMD", "bash", "-c", "cat < /dev/null > /dev/tcp/127.0.0.1/11800" ]
```

Comment:
Why do we switch from `nc` to the bash's built-in tcp file?

Reply:
I met some mistakes in my gitpod server and made some testing changes. But I forgot to launch the pr to the master of my forked repo firstly to run CI，I will close the pr now, so embarrassing...  Sorry~